<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseOpenWindow</class>
 <widget class="QMainWindow" name="DatabaseOpenWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="DataSelectionLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="SpecificTimeLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_5">
          <property name="text">
           <string>Show data for a specific time</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="SpecificTimeSelectionLayout">
          <item>
           <layout class="QVBoxLayout" name="YearLayout">
            <item>
             <widget class="QCheckBox" name="YearCheckbox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Year</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="YearsList">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_8">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="MonthLayout">
            <item>
             <widget class="QCheckBox" name="MonthCheckbox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Month</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="MonthsList">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_7">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="DayLayout">
            <item>
             <widget class="QCheckBox" name="DayCheckbox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Day</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="DaysList">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QRadioButton" name="radioButton_6">
          <property name="text">
           <string>Show a&amp;ll data</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>80</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
